DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/ArcGISProjectUtils.cs

```diff
@@ -0,0 +1,66 @@
+using ArcGIS.Core.Data;
+using ArcGIS.Core.Data.DDL;
+using ArcGIS.Desktop.Core;
+using ArcGIS.Desktop.Framework.Threading.Tasks;
+
+namespace Speckle.Converters.ArcGIS3.Utils;
+
+public class ArcGISProjectUtils
+{
+  public string GetDatabasePath()
+  {
+    string fGdbName = "Speckle.gdb";
```

Comment:
hardcoded? is this always this name? if so can this be a const please, perhaps:

```
private const string FGDB_NAME = "Speckle.gdb";
``` 

Assuming it should be private, assuming the name does not change?